src/features/hotspots/details/TimelinePicker.tsx

```diff
@@ -7,57 +7,58 @@ import { useSpacing } from '../../../theme/themeHooks'
 
 type Props = {
   index?: number
-  onTimelineChanged?: (value: number) => void
+  onTimelineChanged?: (value: number, index: number) => void
 }
 
 const TimelinePicker = ({ index = 0, onTimelineChanged }: Props) => {
   const { t } = useTranslation()
   const spacing = useSpacing()
 
   const data = useMemo(() => {
-    const values = [1, 7, 14, 30]
+    const values = [1, 14, 30]
```

Comment:
I asked Coco what we wanted to do about the graph, and since it was wider now the bars are pretty wide and it looked strange. We decided to remove 7, since you can see the last 7 days pretty easily over the last 14 anyway